doc/classes/Node.xml

```diff
@@ -421,6 +421,12 @@
 				Returns the node's [Viewport].
 			</description>
 		</method>
+		<method name="get_window" qualifiers="const">
+			<return type="Window" />
+			<description>
+				Returns [Window] that contains this node.
```

Comment:
> accurate performance wise though.

For the majority of cases it's the same, if it's called for a nested viewport, it will do few extra calls, but it should not make any practical difference.